src/__tests__/scales/saxon.ts

```diff
@@ -0,0 +1,102 @@
+import { GradeBands } from '../../GradeBands'
+import { Saxon } from '../../scales'
+
+describe('Saxon', () => {
+  describe('Get Score', () => {
+    test('7a > 5', () => {
+      const highGrade = Saxon.getScore('7a')
+      const lowGrade = Saxon.getScore('5')
+      expect(highGrade[0]).toBeGreaterThan(lowGrade[1])
+    })
+
+    test('10c > 8b', () => {
+      const highGrade = Saxon.getScore('10c')
+      const lowGrade = Saxon.getScore('8b')
+      expect(highGrade[0]).toBeGreaterThan(lowGrade[1])
+    })
+
+    test('9b > 9a', () => {
+      const highGrade = Saxon.getScore('9b')
+      const lowGrade = Saxon.getScore('9a')
+      expect(highGrade[0] < lowGrade[1] && highGrade[0] > lowGrade[0])
+      expect(highGrade[1]).toBeGreaterThan(lowGrade[1])
+    })
+
+    test('3 > 2', () => {
+      const highGrade = Saxon.getScore('3')
+      const lowGrade = Saxon.getScore('2')
+      expect(highGrade[0] < lowGrade[1] && highGrade[0] > lowGrade[0])
+      if (lowGrade[1] !== undefined && highGrade[1] !== undefined) {
+        expect(highGrade[1]).toBeGreaterThan(lowGrade[1])
+      }
+    })
+  })
+
+  describe('Invalid grade format', () => {
```

Comment:
@TaliaMalia , please add a check that a grade such as 5a is disallowed.

Reply:
I added a test called "invalid character" that disallows to use grades < 7 to have letters from a to c.
While doing that I recognized that the saxonGradeRegex I wrote was incorrect and did not consider that the grades 1-6 are not allowed to have letters. I changed that now and everything works as it should.